### ISSUE TITLE
No Changelog found (requires manual confirmation)

### ISSUE THREAD

**Issue by coamtaker618** [rickysarraf/apt-offline#92]
TL;DR:
I can't get rid of the requirement to press a button on `apt-offline install` due to the requiring user input at:

> No  changelog  available

Full Description:
I am running the following 4 commands:
`sudo apt-offline set --update --upgrade --generate-changelog $APTOFFLINESET`
`sudo apt-offline get $APTOFFLINESET -d $APTOFFLINETEMP --bundle $APTOFFLINEGET`
`sudo apt-offline install $APTOFFLINEGET`
`sudo apt update`

on two machines and the update is successful, but I am hitting:
> No changelog available
> (Y) Yes. Proceed with installation
> (N) No, Abort.
> (C) Display changelog
> (?) Display this help message.
> What would you like to do next:	 (y, N, ?)y

How can I run `apt-offline` without hitting that prompt? I am very interested in automating these updates & this prompt makes that problematic :(

**Comment by rickysarraf** [comment]
There's a `--skip-changelog` option in there in the `install` command. Does that not work ?

```
rrs@priyasi:~/NoBackup$ apt-offline install -h
usage: apt-offline install [-h] [--verbose] [--simulate]
                           [--install-src-path INSTALL_SRC_PATH]
                           [--skip-bug-reports] [--skip-changelog]
                           [--allow-unauthenticated]
                           apt-offline-download.zip | apt-offline-download/

positional arguments:
  apt-offline-download.zip | apt-offline-download/
                        Install apt-offline data, a bundle file or a directory

optional arguments:
  -h, --help            show this help message and exit
  --verbose             Enable verbose messages
  --simulate            Just simulate. Very helpful when debugging
  --install-src-path INSTALL_SRC_PATH
                        Install src packages to specified path.
  --skip-bug-reports    Skip the bug report check
  --skip-changelog      Skip display of changelog
  --allow-unauthenticated
                        Ignore apt gpg signatures mismatch
09:33 ♒♒♒   ☺ 😄    
```

**Comment by coamtaker618** [comment]
Doh, I missed that >_<   I was so busy looking at forcing a changelog at the set command it never occurred to me to skip it on install!  Just tried it & it works as intended! Thanks!
